webui/react/src/hooks/useDebouncedSettings.ts

```diff
@@ -31,7 +31,7 @@ export function useDebouncedSettings<T extends t.HasProps | t.ExactC<t.HasProps>
   const settingsObs = useMemo(() => userSettings.get(type, path), [type, path]);
   const [localState, updateLocalState] = useState<Loadable<T | null>>(NotLoaded);
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
saw issues where we were still seeing two requests to the search endpoint because settings was loaded but column widths were not. `useObservable` subscribes to the observable in `useLayoutEffect` instead of `useEffect`, so we do that here to ensure all the subscriptions fire at the same time.